### ISSUE TITLE
Add State Management (Pinia) to the Symfony/UX-Vue

### ISSUE THREAD

**Issue by deanilvincent** [symfony/ux#711]
Hi, 

First of all, I'd like to thank you for creating this awesome symfony/ux-vue plugin. Few configurations but has powerful usage especially if I want to a vue.js template and logics within a component in the Symfony Application. I also followed the documentation and completed the setup from here [https://symfony.com/bundles/ux-vue/current/index.html](https://symfony.com/bundles/ux-vue/current/index.html)

My question, is there a way to create a root state and access that state regardless of the components that want to access it? I'm using Pinia as my state management. [https://pinia.vuejs.org/](https://pinia.vuejs.org/).

On a regular vue.js app, we configured the pinia state at the `main.js` file.
```
import { createApp } from "vue";
import { createPinia } from "pinia";
import App from "./App.vue";

const app = createApp(App);
app.use(createPinia());
```

Thank you and I look forward to your response.

**Comment by Kocal** [comment]
Hi, you can use the `vue:before-mount` event, see https://symfony.com/bundles/ux-vue/current/index.html#events

**Comment by deanilvincent** [comment]
> Hi, you can use the `vue:before-mount` event, see https://symfony.com/bundles/ux-vue/current/index.html#events

Thank you! I'll try that. 
Just a quick question, where do we properly declare these line of codes? Is it within the app.js file or somewhere else?
```
document.addEventListener('vue:before-mount', (event) => {
    ...
    app.use(router);
});
```

**Comment by Kocal** [comment]
Yep, inside the  app.js  file

**Comment by Kocal** [comment]
Can be closed since #712 has been merged
